### PR TITLE
Square feature

### PR DESCRIPTION
**Pull request by alexanderludvigs**

### CODE REVIEW

File: build/reports/tests/classes/is.ru.TicTacToe.WorldTest.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
 <input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
 </label>
 </div>Generated by 
-<a href="http://www.gradle.org">Gradle 2.8</a> at Nov 1, 2015 6:01:25 PM</p>
+<a href="http://www.gradle.org">Gradle 2.8</a> at Nov 1, 2015 6:38:39 PM</p>
```
Comment (tinkrtailor) [inline build/reports/tests/classes/is.ru.TicTacToe.WorldTest.html:92]: I though we added the build  folder to  our .gitignore

File: src/main/java/is/task/TicTacToe/Square.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package main.java.is.task.TicTacToe;
+
+public class Square {
+   
+    protected int squareNumber;
```
Comment (asdissvava) [inline src/main/java/is/task/TicTacToe/Square.java:5]: Why do you have this?

File: src/main/java/is/task/TicTacToe/Square.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    // The current marked symbol in the square
+    public char getCurrentSquareMark(){
+        return this.currentSquareMark;
+    }
+
+    public void setCurrentSquareMark(char mark){
+        this.currentSquareMark = mark;
+    }
+
+}
```
Comment (tinkrtailor) [inline src/main/java/is/task/TicTacToe/Square.java:48]: This class seems pretty straight forward I don't see anything wrong with it

File: src/test/java/is/task/TicTacToe/SquareTest.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    }
+
+    @Test
+    public void testSetCurrentSquareMark_CurrentSquareMarkIsO_returnO() {
+        Square s = new Square();
+        s.setCurrentSquareMark('O');
+
+        assertEquals('O', s.getCurrentSquareMark());
+    }
+}
```
Comment (tinkrtailor) [inline src/test/java/is/task/TicTacToe/SquareTest.java:63]: This looks like a lot of code coverage

**Comment by tinkrtailor** [comment]
We should have left out the build folders but it seems okay

**Pull request merged**